java/BUILD

```diff
@@ -14,10 +14,25 @@ DISTRO_VERSIONS = {
     "_debian10": versions_amd64_debian10,
 }
 
+USERS = [
+    (
+        "",
+        "_root",
+        "0",
+        "/",
+    ),
+    (
+        "_nonroot",
+        "_nonroot",
+        "1002",
+        "/home/nonroot",
```

Comment:
I don't think you need to re-define these. They are defined in the lower base images.

```
$ git grep NONROOT
base/BUILD:load(":base.bzl", "NONROOT", "distro_components")
base/BUILD:    gid = NONROOT,
base/BUILD:    uid = NONROOT,
base/BUILD:    gid = NONROOT,
base/base.bzl:NONROOT = 65532
base/base.bzl:        for (user, uid, workdir) in [("root", 0, "/"), ("nonroot", NONROOT, "/home/nonroot")]:
experimental/python3/BUILD:load("//base:base.bzl", "NONROOT")
experimental/python3/BUILD:        user = "%d" % NONROOT,
```

I expected something like these:

https://github.com/GoogleContainerTools/distroless/blob/40976b8b65aec783a7571ad20d7b1fa0cd12c69e/cc/BUILD#L12
https://github.com/GoogleContainerTools/distroless/blob/40976b8b65aec783a7571ad20d7b1fa0cd12c69e/cc/BUILD#L24-L27

That is, I think it's unnecessary to set `user` and `workdir` below?